docs/modules/graphics.md

```diff
@@ -24,6 +24,12 @@ This is the name of the default font used for `Canvas.print(str, x, y, color)`.
 This is the height of the canvas/viewport, in pixels.
 #### `static width: Number`
 This is the width of the canvas/viewport, in pixels.
+#### `static offsetX : Number`
```

Comment:
I don't think we should document offsetX and offsetY as part of the public API, unless you can write values to them as well.

Reply:
Yep those shouldn't have been there. Fixed it.